packages/node_modules/@webex/plugin-authorization-browser-first-party/test/unit/spec/authorization.js

```diff
@@ -86,7 +94,7 @@ browserOnly(describe)('plugin-authorization-browser-first-party', () => {
 
           return webex.authorization.when('change:ready')
             .then(() => {
-              assert.calledOnce(webex.request);
+              assert.calledTwice(webex.request);
```

Comment:
This test had been broken since september 2019. The addition of the call to get the catalog first had resulted in two calls to webex.request instead of only one.